config/genesis.go

```diff
@@ -0,0 +1,82 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+
+	apiConfig "github.com/spacemeshos/go-spacemesh/api/config"
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/hash"
+	"github.com/spacemeshos/go-spacemesh/log"
+)
+
+const (
+	DefaultGenesisConfigFileDir  = "data"
+	DefaultGenesisConfigFileName = "genesis.conf"
+	defaultGenesisExtraData      = "mainnet"
+
+	DefaultTestGenesisTime      = "2022-12-25T00:00:00+00:00"
```

Comment:
where do we use this? should be possible to cleanup

Reply:
moved it to `cmd/node/node_test.go`